NCTRC/frontend/tests/handlers.test.ts

```diff
@@ -122,33 +122,39 @@ test("Sign in and sign out already existing user", async () => {
 
 test("Create a user to be rejected via fever/COVID question", async () => {
   await updateMaxCapacity(updateMaxCapacityRequestModel111);
-  await createAndSigninUser(rejectTempNewUserRequestModel).then((result: any) => {
-    expect(result.statusCode).toBe(412);
-  }).catch((err: any) => {
-    console.error(err);
-    fail();
-  }); 
-  expect(await checkUserExists(userRequestModel)).toBe(true);
-  expect(await deleteUser(userRequestModel)).toBe(200);
-  expect(await checkUserExists(userRequestModel)).toBe(false);
-  await createAndSigninUser(rejectTempNewUserRequestModel).then((result: any) => {
-    expect(result.statusCode).toBe(412);
-  }).catch((err: any) => {
-    console.error(err);
-    fail();
-  }); 
+  createAndSigninUser(rejectTempNewUserRequestModel)
```

Comment:
add an await here - this is my bad. Without the await it introduces a race condition